qiskit/visualization/timeline/generators.py

```diff
@@ -0,0 +1,410 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+A collection of functions that generate drawing objects from formatted input data.
+See :py:mod:`~qiskit.visualization.timeline.types` for the detail of input data.
+
+Adding a custom generator:
+    The function in this module are generators for drawing objects.
```

Comment:
```suggestion
    The function in this module are generators for drawing objects.
```
`drawing objects` is a slightly ambiguous name because it sounds like "this generator draws Objects" rather than "these are the generators for DrawingObjects". I think `DrawingObjects` are drawn objects, not objects that draw. Not sure yet what a better name might be. Perhaps there isn't a better name.